boards/nxp/imx95_evk/doc/index.rst

```diff
@@ -0,0 +1,262 @@
+.. _imx95_evk:
+
+NXP i.MX95 EVK
+##############
+
+Overview
+********
+
+The i.MX95 EVK (IMX95LPD5EVK-19) board is a platform designed to show the
+most commonly used features of the i.MX 95 automotive applications processor.
+It is an entry-level development board, which helps developers to get familiar
+with the processor before investing a large amount of resources in more
+specific designs. The i.MX95 device on the board comes in a compact
+19 x 19 mm package.
+
+Hardware
+********
+
+- i.MX 95 automotive applications processor
+
+  - The processor integrates up to six Arm Cortex-A55 cores, and supports
+    functional safety with built-in Arm Cortex-M33 and -M7 cores
+
+- DRAM memory: 128-Gbit LPDDR5 DRAM
+- eMMC: 64 GB Micron eMMC
+- SPI NOR flash memory: 1 Gbit octal flash memory
+- USB interface: Two USB ports: Type-A and Type-C
+- Audio codec interface
+
+  - One audio codec WM8962BECSN/R with one TX and RX lane
+  - One 3.5 mm 4-pole CTIA standard audio jack
+  - One 4-pin connector to connect speaker
+
+- Ethernet interface
+
+  - ENET2 controller
+
+    - Connects to a 60-pin Ethernet connector
+    - Supports Ethernet PHY daughter cards that can be configured to operate
+      at 100 Mbit/s or 1000 Mbit/s
+
+  - ENET1 controller
+
+    - Supports 100 Mbit/s or 1000 Mbit/s RGMII Ethernet with one RJ45
+      connector connected with an external PHY, RTL8211
+
+  - 10 Gbit Ethernet controller
+
+    - Supports XFI and USXGMII interfaces with one 10 Gbit RJ45 ICM connected
+      with an external PHY, Marvell AQR113C
+
+- M.2 interface: One Wi-Fi/Bluetooth Murata Type-2EL module based on NXP AW693
+  chip supporting 2x2 Wi-Fi 6 and Bluetooth 5.2
+
+- MIPI CSI interface: Connects to one 36-pin miniSAS connector using x4 lane
+  configuration
+- MIPI CSIDSI interface: Connects to one 36-pin miniSAS connector using x4 lane
+  configuration
+- LVDS interface: two mini-SAS connectors each with x4-lane configuration
+- CAN interface: Two 4-pin CAN headers for external connection
+- SD card interface: one 4-bit SD3.0 microSD card
+- I2C interface: I2C1 to I2C7 controllers
+- FT4232H I2C interface: PCT2075 temperature sensor and current monitoring devices
+- DMIC interface: two digital microphones (DMIC) providing a single-bit PDM output
+- ADC interface: two 4-channel ADC header
+- Audio board interface
+
+  - Supports PCIe x4 slot for Quantum board connection
+  - Supports PCIe x8 slot for Audio I/O board connection
+
+- Debug interface
+
+  - One USB-to-UART/MPSSE device, FT4232H
+  - One USB 2.0 Type-C connector (J31) for FT4232H provides quad serial ports
+
+Supported Features
+==================
+
+The Zephyr imx95_evk_m7 board configuration supports the following hardware features:
```

Comment:
this is not the board name at all, put \`\` around it, put the actual board name you supply to west and remove board configuration because such a term has never existed in zephyr and replace with board target